### ISSUE TITLE
Issue parsing basic React classes

### ISSUE THREAD

**Issue by vedran** [i18next/i18next-scanner#136]
Hello. Thanks for all of your effort in creating and maintaining this library.

I have run into two issues parsing a basic class and have stripped it down to its most basic repro state. Due to the simple nature of these repro cases, my assumption is that I've done something wrong in my configuration...

### 1. Empty class case:

```js
import React from 'react'

class MyComponent extends React.Component {
}
```

Results in an error that points to the period in `React.Component`
```
i18next-scanner: Unable to parse I18n component with the content
{ SyntaxError: Unexpected token (3:31)
at Object.pp$4.raise (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:2844:13)
    at Object.pp.unexpected (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:690:8)
    at Object.pp.expect (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:684:26)
    at Object.pp$1.parseClass (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:1315:8)
    at Object.parseClass (/Users/code/myproject/node_modules/acorn-class-fields/index.js:49:29)
    at Object.parseClass (/Users/code/myproject/node_modules/acorn-static-class-features/index.js:49:29)
    at Object.parseClass (/Users/code/myproject/node_modules/acorn-private-methods/index.js:38:29)
    at Object.pp$1.parseStatement (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:839:17)
    at Object.parseStatement (/Users/code/myproject/node_modules/acorn-stage3/node_modules/acorn-dynamic-import/lib/index.js:63:118)
    at Object.parseStatement (/Users/code/myproject/node_modules/acorn-import-meta/index.js:38:22) pos: 58, loc: Position { line: 3, column: 31 }, raisedAt: 59 }
```

### 2. Simple render case

```js
import React from 'react'

class MyComponent {
  render() {
    <div>
      Hello
    </div>
  }
}
```

Results in an error at the opening parenthesis of `render()`:

```
i18next-scanner: Unable to parse I18n component with the content
{ SyntaxError: Unexpected token (4:8)
at Object.pp$4.raise (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:2844:13)
    at Object.pp.unexpected (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:690:8)
    at Object.pp.expect (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:684:26)
    at Object.pp$1.parseClass (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:1315:8)
    at Object.parseClass (/Users/code/myproject/node_modules/acorn-class-fields/index.js:49:29)
    at Object.parseClass (/Users/code/myproject/node_modules/acorn-static-class-features/index.js:49:29)
    at Object.parseClass (/Users/code/myproject/node_modules/acorn-private-methods/index.js:38:29)
    at Object.pp$1.parseStatement (/Users/code/myproject/node_modules/i18next-scanner/node_modules/acorn/dist/acorn.js:839:17)
    at Object.parseStatement (/Users/code/myproject/node_modules/acorn-stage3/node_modules/acorn-dynamic-import/lib/index.js:63:118)
    at Object.parseStatement (/Users/code/myproject/node_modules/acorn-import-meta/index.js:38:22) pos: 58, loc: Position { line: 3, column: 31 }, raisedAt: 59 }
```

#### Version
- i18next: <i>13.1.5</i>
- i18next-scanner: <i>2.9.2</i>

#### Configuration
```js
module.exports = {
  src: [
    'src/**/*.{html,js,jsx}',
  ],
  dest: './',
  options: {
    debug: true,
    removeUnusedKeys: false,
    sort: false,
    lngs: ['en'],
    func: {
      list: ['t', 'i18next.t'],
      extensions: ['.js']
    },
    trans: {
      component: 'I18n',
      i18nKey: 'i18nKey',
      extensions: ['.js'],
      fallbackKey: function (ns, value) {
        return value;
      }
    },
    defaultValue: (lng, ns, key) => {
      if (lng === 'en') {
        return key; // Use key as value for base language
      }
      return ''; // Return empty string for other languages
    },
    ns: [
      'resource' // default
    ],
    defaultNs: 'resource',
    resource: {
      loadPath: 'app/i18n/{{lng}}/{{ns}}.json',
      savePath: 'app/i18n/{{lng}}/{{ns}}.json', // or 'app/i18n/${lng}/${ns}.saveAll.json'
      jsonIndent: 4
    },
    nsSeparator: ':', // namespace separator
    keySeparator: '.', // key separator
    plural: true, 
    interpolation: {
      prefix: '{{',
      suffix: '}}'
    }
  },
};
```

**Comment by cheton** [comment]
Fixed in 2.10.0

**Comment by vedran** [comment]
Thanks!
